community/calcurse/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+sed -i test/Makefile.in -e '/ical-009.sh \\/d'
+
+./configure \
+    --disable-docs \
+    --without-asciidoc \
+    --prefix=/usr \
+    --mandir=/usr/share/man \
+    --sysconfdir=/etc \
```

Comment:
We typically put the directories above the options.

Reply:
```sh
./configure \
    --prefix=/usr \
    etc \
    etc
```

Reply:
I may have to read the style guide indepth this time. 
Thank you again @dylanaraps